### PR TITLE
Minor improvements to `plantuml` doc generator

### PR DESCRIPTION
**Pull request by wing328**
- Minor improvements to `plantuml` doc generator (such as copyright note, mark as beta, use OAS 3, etc)

cc @pburls

### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [x] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [ ] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

File: bin/windows/plantuml-documentation-petstore.bat [hunk 1]
```diff
@@ -5,6 +5,6 @@ If Not Exist %executable% (
 )
 
 REM set JAVA_OPTS=%JAVA_OPTS% -Xmx1024M -DloggerPath=conf/log4j.properties
-set ags=generate  --artifact-id "plantuml-petstore-documentation" -i modules\openapi-generator\src\test\resources\2_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
+set ags=generate -t modules\openapi-generator\src\main\resources\plantuml-documentation -i modules\openapi-generator\src\test\resources\3_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
```
Comment (pburls) [inline bin/windows/plantuml-documentation-petstore.bat:8]: Think you need to remove the `-documentation` suffix from this file too.
```suggestion
set ags=generate -t modules\openapi-generator\src\main\resources\plantuml -i modules\openapi-generator\src\test\resources\3_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
```

Comment (wing328) [reply bin/windows/plantuml-documentation-petstore.bat]: Fixed via https://github.com/OpenAPITools/openapi-generator/pull/6298/commits/851b94a3c800e69fc374797a024236ccd06b2236

**Comment by wing328** [comment]
@pburls is https://twitter.com/pburls your twitter account? If yes, I'll tag you in a tweet to promote the new generator.

**Comment by pburls** [comment]
> @pburls is https://twitter.com/pburls your twitter account? If yes, I'll tag you in a tweet to promote the new generator.

@wing328 Yes, that's me. Thanks 😊

**Comment by wing328** [comment]
Sent https://twitter.com/oas_generator/status/1261325614549577735

**Pull request merged**